packages/clay-date-picker/src/DayNumber.tsx

```diff
@@ -13,26 +13,36 @@ interface IProps {
 	daySelected: Date;
 	disabled?: boolean;
 	onClick: (date: Date) => void;
+	onKeyUp?: (event: React.KeyboardEvent) => void;
+	onKeyDown?: (event: React.KeyboardEvent) => void;
 }
 
+const noop = () => {};
+
+const KEY_CODE_SPACE = 32;
+
 const ClayDatePickerDayNumber: React.FunctionComponent<IProps> = ({
 	day,
 	daySelected,
 	disabled,
-	onClick,
+	onClick = noop,
+	onKeyDown = noop,
+	onKeyUp = noop,
```

Comment:
I think we don't need to keep adding `noop` ​​all the time here, I would avoid that. Since this is a component that is only used in the DatePicker, we don't need noop. We may not even need `onKeyDown` and `onKeyUp` for this case since we are not using it. If this were a public component it would make more sense.

Reply:
Yes, makes sense Matu. Since We're not exposing directly this component, I'll remove these noops. :)

Reply:
> We may not even need onKeyDown and onKeyUp for this case since we are not using it. If this were a public component it would make more sense.

So, Should I remove from props interface?

Reply:
Removed after some internal conversations